### ISSUE TITLE
Snap install, can't pin it in Plank or find it in Slingscold, Fix included

### ISSUE THREAD

**Issue by constantinosgeorgiou** [klaussinani/tusk#188]
## __tldr: It's not actually a bug, I provide a fix at the end.__
**Describe the bug**
First time using snap so I might be doing something wrong...
Once I installed Tusk successfully via terminal I wasn't able to pin Tusk to my dock (Plank) nor find it in Slingscold.

**To Reproduce**
Install it via snap in terminal.
````
snap install tusk
````
**Expected behavior**
It works great!
The only problem is that __tusk.desktop__ is not located in _/usr/share/applications_ but _/snap/tusk/current/meta/gui_

**Technical Info**
 - OS: Mint 19.1 tessa
 - Tusk Version: 0.22.0

### How to fix it
- Copy _tusk.desktop_ to /usr/share/applications
    ````
    sudo cp /snap/tusk/current/meta/gui/tusk.dekstop /usr/share/applications
    ````
- Edit tusk.desktop located in /usr/share/applications
    ````
    sudo nano /usr/share/applications/tusk.desktop
    ````
- Change the Exec line to:
    ````
    Exec=/snap/tusk/current/tusk
    ````
- Change the Icon line to:
    ````
    Icon=/snap/tusk/current/meta/gui/icon.png
    ````
- Make tusk.desktop located in /usr/share/applications executable
- Run it
- Pin it
- :cookie: 

Because it's my first time trying out snap, it took me a couple of minutes to figure things out...
If you think I'm doing something wrong feel free to correct me!
If there's a better way to fix this please let me know :unicorn:

**Comment by Tzion0** [comment]
For me, probably not a perfect way, i simply do this and it worked:
```
cp /var/lib/snapd/desktop/applications/<app>.desktop /usr/share/applications/
```
- Run it
- Pin it

**Comment by constantinosgeorgiou** [comment]
I guess it should work. I no longer use Mint. Closing this issue as it is quite old.
